mods/ToxicPlayer@Flushtro/meta.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "Flushtro",
+  "requires-steamodded": true,
+  "requires-talisman": false,
+  "categories": ["Joker"],
+  "author": "ToxicPlayer",
+  "repo": "https://github.com/N0tToxicPlayer/Flushtro",
+  "downloadURL": "https://github.com/N0tToxicPlayer/Flushtro/releases/download/v1.10/flushtro.zip",

```

```suggestion
  "downloadURL": "https://github.com/N0tToxicPlayer/Flushtro/releases/latest/download/flushtro.zip",
```

since you have `automatic-version-check: true` then your downloadURL should point to the latest release instead of a specific tag, this means that whenever you make new releases and upload `flushtro.zip` in them this link will be a valid link to the newest one